### PR TITLE
add maf; reinstate tests

### PR DESCRIPTION
**Pull request by deepchatterjeeligo**

### CODE REVIEW

File: amplfi/architectures/pyproject.toml [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
-authors = ["Ethan Jacob Marx <ethan.marx@ligo.org>"]
+authors = ["Ethan Jacob Marx <ethan.marx@ligo.org>", "Deep Chatterjee <deep.chatterjee@ligo.org>"]
 packages = [{include = "amplfi"}]
 
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-pyro-ppl = "^1.9.0"
+ml4gw = {path = "../../ml4gw", develop = true}
+pyro-ppl = "*"
```
Comment (EthanMarx) [inline amplfi/architectures/pyproject.toml:12]: Is there a reason we're unconstraining `pyro`? I imagine there's _some_ version where our code will break.

**Review by EthanMarx — COMMENTED** [review]
Looks good. Thanks for adding the testing infra back in. Just a question about version constraints for pyro (and pytest as well)

File: amplfi/architectures/pyproject.toml (hunk 1 above)
Comment (deepchatterjeeligo) [reply amplfi/architectures/pyproject.toml:12]: Usually we should constrain a version when a feature breaks, and mention the issue from the corresponding repo. So far I have not found our pyro features to break.

Comment (deepchatterjeeligo) [reply amplfi/architectures/pyproject.toml:12]: But at the same time, I don't feel very strongly.

Comment (deepchatterjeeligo) [reply amplfi/architectures/pyproject.toml]: done

**Review by EthanMarx — APPROVED** [review]

**Pull request merged**